reference/sisdata.v1.yaml

```diff
@@ -325,11 +360,11 @@ components:
         startDate:
           type: string
           format: date
-          description: Start of this subject for the student. Date cannot be in the future.
+          description: Start of this subject for the student. (ISO8601)
         endDate:
           type: string
           format: date
-          description: Start of this subject for the student. Date cannot be in the future. null if not ended
+          description: Start of this subject for the student. (ISO8601). null if not ended
```

Comment:
* End of this subject for the student

Reply:
Wat bedoelen we hier eigenlijk met _null if not ended?_ null if no enddate is given?

Reply:
null if you do not know the end date, so it is until end of schoolyear?